agent/src/main/java/com/microsoft/applicationinsights/agent/internal/agent/AgentImplementation.java

```diff
@@ -136,24 +135,28 @@ private static void appendJarsToBootstrapClassLoader(Instrumentation inst) throw
     }
 
     public static String getAgentJarLocation() throws UnsupportedEncodingException {
+        ClassLoader systemClassLoader = ClassLoader.getSystemClassLoader();
         try {
-            ClassLoader systemClassLoader = ClassLoader.getSystemClassLoader();
             if ((systemClassLoader instanceof URLClassLoader)) {
                 for (URL url : ((URLClassLoader)systemClassLoader).getURLs()) {
                     String urlPath = url.getPath();
+
                     if (urlPath.indexOf(AGENT_JAR_PREFIX) != -1) {
+                        InternalAgentLogger.INSTANCE.logAlways(InternalAgentLogger.LoggingLevel.INFO,"Agent jar found at " + urlPath);
                         int index = urlPath.lastIndexOf('/');
                         urlPath = urlPath.substring(0, index + 1);
                         return urlPath;
                     }
+
                 }
             }
         } catch (Throwable throwable) {
+            InternalAgentLogger.INSTANCE.logAlways(InternalAgentLogger.LoggingLevel.ERROR,"Unable to find the Agent Jar");
```

Comment:
Remove this extra log message. It's going to the same logger, same level and there's no new information here. Feel free to update the wording in the log message on the next line.

Reply:
okay thanks will do this.